### ISSUE TITLE
Re-authorization

### ISSUE THREAD

**Issue by CyberParadise** [yisbug/whatsapp-nodejs#5]
what about reauthorization? 
i get "status":"ok","type":"existing"
and if you delete the record from the database, then a new device will be generated during registration

**Comment by cobra2k5** [comment]
> what about reauthorization? i get "status":"ok","type":"existing" and if you delete the record from the database, then a new device will be generated during registration
> 
> off topic: add addresses of crypto wallets for donations in readme

I GitHub sponsorship would also be nice.

**Comment by yisbug** [comment]
> 

Good idea, I have added the donation address to readme, thank you very much for your support. @CyberParadise @cobra2k5
